etna/core/models/settings.py

```diff
@@ -12,7 +13,37 @@
 
 @register_setting(icon="list-ul")
 class SiteSettings(BaseSetting, ClusterableModel):
+    beta_banner_standfirst = models.CharField(
+        max_length=200,
+        default="What does the 'beta' label mean?",
+        verbose_name="standfirst",
+    )
+    beta_banner_link = models.CharField(
+        max_length=200, blank=True, verbose_name="link to"
+    )
+    beta_banner_link_text = models.CharField(
+        max_length=200,
+        default="Find out more",
+        verbose_name="link text",
+    )
+    beta_banner_text = RichTextField(
+        verbose_name="text",
+        default=(
+            "<p>This beta site contains new services and features currently in development. Many of these features are "
+            "works in progress and are being updated regularly. You can help us improve them by providing feedback as you "
+            "use the site.</p> "
+        ),
+    )
     panels = [
+        MultiFieldPanel(
+            heading="BETA banner",
+            children=[
+                FieldPanel("beta_banner_standfirst"),
+                FieldPanel("beta_banner_link"),
+                FieldPanel("beta_banner_link_text"),
+                FieldPanel("beta_banner_text"),
```

Comment:
@mozaki1 One last/tiny thing: I find it helps editors to see fields in the order they are used in the output. Since the link/button comes at the end, a better order might be:
```suggestion
                FieldPanel("beta_banner_text"),
                FieldPanel("beta_banner_link"),
                FieldPanel("beta_banner_link_text"),
                
```